debezium-quarkus-outbox/integration-tests/src/test/java/io/debezium/outbox/quarkus/it/DatabaseTestResource.java

```diff
@@ -20,29 +20,26 @@
  */
 public class DatabaseTestResource implements QuarkusTestResourceLifecycleManager {
 
-    private static final String POSTGRES_USER = "postgres";
-    private static final String POSTGRES_PASSWORD = "postgres";
-    private static final String POSTGRES_DBNAME = "postgres";
-    private static final String POSTGRES_IMAGE = "debezium/postgres:9.6";
-    private static final DockerImageName POSTGRES_DOCKER_IMAGE_NAME = DockerImageName.parse(POSTGRES_IMAGE)
-            .asCompatibleSubstituteFor("postgres");
+    private static PostgresInfrastructure POSTGRES_INFRASTRUCTURE;
 
-    private static PostgreSQLContainer<?> container;
+    private static PostgreSQLContainer<?> postgresContainer;
+
+    public static void setPostgresInfrastructure() {
+        POSTGRES_INFRASTRUCTURE = PostgresInfrastructure.getDebeziumPostgresDefaultInfrastructure();
+    }
 
     @Override
     public Map<String, String> start() {
         try {
-            container = new PostgreSQLContainer<>(POSTGRES_DOCKER_IMAGE_NAME)
+            setPostgresInfrastructure();
+            postgresContainer = POSTGRES_INFRASTRUCTURE.getPostgresContainer()
                     .waitingFor(Wait.forLogMessage(".*database system is ready to accept connections.*", 2))
-                    .withUsername(POSTGRES_USER)
-                    .withPassword(POSTGRES_PASSWORD)
-                    .withDatabaseName(POSTGRES_DBNAME)
                     .withEnv("POSTGRES_INITDB_ARGS", "-E UTF8")
                     .withEnv("LANG", "en_US.utf8")
                     .withStartupTimeout(Duration.ofSeconds(30));
 
-            container.start();
-            return Collections.singletonMap("quarkus.datasource.jdbc.url", container.getJdbcUrl());
+            postgresContainer.start();
```

Comment:
Starting and stopping the same container instance might slow tests down. I think this can be replaced with a reset hook like we do in the connector test cases. Or is there a reason @jpechane ?

Reply:
A Quarkus test resource is started/stopped once for the entire suite run. Does this address your concern, @rk3rn3r?